cb_hr_views/security/hr_attendance_security.xml

```diff
@@ -8,6 +8,10 @@
         <field name="name">Employee</field>
     </record>
 
+    <record id="manual_attendance_group" model="res.groups">
+        <field name="name">Manual Attendance</field>
```

Comment:
Change for Manual Attendance Menu please

Reply:
Changed!